docs/integrations/gitlab/locations.md

```diff
@@ -21,6 +21,18 @@ integrations:
 
 :::note Note
 
+When using the [`backstage.io/source-location`](../../features/software-catalog/well-known-annotations.md#backstageiosource-location): `url:https://{gitlab host}/{group}/{subgroup}/{repository}/` annotation to create a component, you must include `/blob/main/` in the URL path. For example:
```

Comment:
Heya! 👋 

Sorry for not getting around to this sooner, we use Gitlab internally and we've never had to manually set the `backstage.io/source-location`, as far as I can see it does automatically add the `/blob/main/` parts when you import something via the catalog.

I'm just curious how this came about, because I also see you mention that without this, the _**README**_ section of the component page will fail to load, what does this refer to? Just trying to puzzle it together 😅 

LMK! 👏